Sprintfinity3902/Sprites/Link/LinkDownSprite.cs

```diff
@@ -28,6 +29,9 @@ public LinkDownSprite(Texture2D texture)
             Animation.AddFrame(Sprite1, 1 / 5f);
         }
 
-
+        public override void Draw(SpriteBatch spriteBatch, Vector2 position, Color color)
+        {
+            base.Draw(spriteBatch, position, color);
+        }
```

Comment:
I don't think this does anything, might just be a leftover.

Reply:
Thanks totally right! I'll fix this.